code/LINDA/LINDA_turf_tile.dm

```diff
@@ -207,7 +207,23 @@
 				if(excited_group)
 					last_share_check()
 
-	air.react()
+	if(air)  //How did this even break?
```

Comment:
I understand where you are coming from, but it seems to happen a great deal and it can bring down the server as happened twice in two rounds.

![image](https://cloud.githubusercontent.com/assets/8165455/26030196/6e79af56-3819-11e7-9dfb-ce477cb3521d.png)